packages/library-base/src/commonMain/kotlin/io/realm/kotlin/internal/Converters.kt

```diff
@@ -97,78 +128,130 @@ internal abstract class PassThroughPublicConverter<T> : CompositeConverter<T, T>
 public inline fun passthrough(value: Any?): Any? = value
 
 // Static converters
-internal object StaticPassThroughConverter : PassThroughPublicConverter<Any>()
+internal object StaticPassThroughConverter : PassThroughPublicConverter<Any?>() {
```

Comment:
This was just done because we didn't need to differentiate. Since all transport types now requires explicit conversion we should delete this and make explicit converts for each type.